### DISCUSSION TITLE
Excessive use of remote commands

### DISCUSSION THREAD

**Discussion by ncot-tech** [flobz/psa_car_controller#620] [Q&A]
I've been using this for well over a year through the Home Assistant integration and it's been fine. I've just gone into HA and noticed the state of charge of my car is unavailable. It's done this before because the Stellantis servers aren't that great and sometimes it all breaks, I think we all know how poor quality the phone app can be. Usually when this happens I open my phone app, confirm that's broken too and ignore it.

I opened my phone app and on the screen it now shows "Excessive use of remote commands" and tapping on that shows me

"The use of your connected services is intensive and may shorten the life of some components and the 12v battery of your vehicle. To ensure the longevity of these, we advise you not to exceed an average of 15 remote commands per day"

Is this something new they've switched on (possibly trying to hide busy servers behind "you might damage your car" warnings to scare us off) or have I managed to mess up my system?

I do have a "charge the car to 80%" automation that pings the car every 5 minutes during charging to find out when it does get to 80%, but beyond that I've not changed anything and besides it's been working fine for literal months without any odd warnings in the phone app.

Anyone else seen this message?

**Comment by flix-one** [comment]
I've got the same error since yesterday. Maybe Peugeot has throttled their API (even more)?
